README.md

```diff
@@ -206,6 +209,19 @@ async function test() {
   const payload = await verifyWithPromise(token)
   // => { a: 1, b: 2, c: 3, iat: 1579521212 }
 }
+
+// custom errorCacheTTL verifier
+const verifier = createVerifier({
+  key: 'secret',
+  cache: true,
+  errorCacheTTL: (tokenError) => {
+    // customize the ttl based on the error code
+    if (tokenError?.code === 'FAST_JWT_KEY_FETCHING_ERROR') {
```

Comment:
where would a user get this error code from? how do they even know they exist? maybe we should expose them?